src/tools/miri/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![feature(cell_update)]
 #![feature(const_option)]
 #![feature(float_gamma)]
+#![cfg_attr(not(bootstrap), feature(float_mul_add_fast))]
```

Comment:
I don't think it is necessary to add this to Miri? It doesn't seem used.

Reply:
I removed the new feature from this PR (which now has only the intrinsics).